aea/helpers/base.py

```diff
@@ -755,3 +756,23 @@ def __eq__(self, other):
             and self.not_before == other.not_before
             and self.save_path == other.save_path
         )
+
+
+def compute_specifier_from_version(version: Version) -> str:
```

Comment:
I feel like I saw this code somewhere else before...worth checking we're not replicating :)